public/scripts/tags.js

```diff
@@ -2324,14 +2332,6 @@ export function initTags() {
         }
     });
 
-    // Initialize auto sort setting based on whether it was sorted before
-    if (power_user.auto_sort_tags === undefined || power_user.auto_sort_tags === null) {
-        power_user.auto_sort_tags = !isManuallySorted();
-        if (power_user.auto_sort_tags) {
-            printCharactersDebounced();
-        }
-    }
-
```

Comment:
Are you 100% sure this can be removed?
I remember a re-print was needed, because any time the tag sorting is changed (even if not defined before), the char list needs an update on in which order the tags are printed on the characters inline.
Otherwise you might have chars in the char list printed with tags in a different order than how they are sorted in the tag management.

If we can confirm the order is the same anyway, as "auto sort" is now the new default print on any char list print if no setting is specified, that's okay.